components/eamxx/cime_config/namelist_defaults_scream.xml

```diff
@@ -285,8 +285,14 @@ be lost if SCREAM_HACK_XML is not enabled.
       <number_of_subcycles hgrid="ne0np4_conus_x4v1_lowcon">5</number_of_subcycles>
     </mac_aero_mic>
 
+    <cosp inherit="atm_proc_base">
+      <cosp_subcolumns>10</cosp_subcolumns>
+      <!-- Frequency at which to call COSP; positive values interpreted as number of steps, negative as number of hours -->
```

Comment:
Can we avoid this CAM-like way of passing parameters? I would rather have something like
```
<cosp_frequency>1</cosp_frequency>
<cosp_frequency_units>nhours</cosp_frequency_units>
```

Reply:
Yes, I think I have this implemented right now. Currently supported `cosp_frequency_units` are “steps” and “hours” now. I don’t really see needing any additional units, and throw an error if units are not steps or hours.